subteams/gs1.md

```diff
@@ -0,0 +1,15 @@
+The GS1 subteam is responsible for discussion, review, and decision making on
+topics related to GS1 standards support, including RFCs.
+
+| Name | GitHub | RocketChat |
+| --- | --- | --- |
+| Adeeb Ahmed | adeebahmed | adeebahmed |
+| Dave Cecchi | davececchi | cecchi |
+| Gena Morgan | | Gena |
+| James Mitchell | jsmitchell | jsmitchell |
+| Matt Perron | MattP007 | mattp |
+| Melanie Nuce | | MelanieNuce |
```

Comment:
My github id is auntmelmel

Reply:
Please add me to the GS1 subteam @jamesmlynch